--- a/deluge/ui/gtkui/systemtray.py	2019-02-25 17:43:19.106392398 -0500
+++ b/deluge/ui/gtkui/systemtray.py	2019-02-25 17:43:49.377024977 -0500
@@ -135,7 +135,7 @@ class SystemTray(component.Component):
                     common.get_logo(32))
             else:
                 try:
-                    self.tray = gtk.status_icon_new_from_icon_name("deluge-panel")
+                    self.tray = gtk.status_icon_new_from_icon_name("deluge")
                 except:
                     log.warning("Update PyGTK to 2.10 or greater for SystemTray..")
                     return
